.jenkins/Jenkinsfile

```
@@ -1,7 +1,9 @@
 def hipBuildTest(String backendLabel) {
         node(backendLabel) {
           stage("SYNC - ${backendLabel}") {
-
+            // Delete the workspace
+            sh "rm ${env.WORKSPACE}/* -fr"
```

Using consistent coding style?
```suggestion
            sh "rm -fr ${env.WORKSPACE}/*"
```

---

Note that this might not delete files and directories starting with '.', which might be an issue.